### PR TITLE
Update being_afraid_of_other_peoples_code.md

### PR DESCRIPTION
**Pull request by nleduc**
Simple English fixes

### CODE REVIEW

File: sections/en/being_afraid_of_other_peoples_code.md [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+The same goes for hundreds of Open Source projects around the world that were eventually abandoned by the original programmer(s) only to be picked up by other skillful programmers. Many of these projects had very little documentation (if any at all), no comments in the codebase, and no guidelines or help to offer at all.
 
-The entire PHP codebase is done in C, a pure procedural programming language, without the use of any kind of framework what so ever.
+The entire PHP codebase is done in C, a pure procedural programming language, without the use of any kind of framework whatsoever.
 
-Whenever you define a class in PHP or whenever you fire up that favorite PHP framework of yours, your running on someones else's pure procedural work!
+Whenever you define a class in PHP or whenever you fire up that favorite PHP framework of yours, you're running on someones else's pure procedural work!
```
Comment (QWp6t) [inline sections/en/being_afraid_of_other_peoples_code.md]: someones -> someone

> ...you're running on **someone** else's...

File: sections/en/being_afraid_of_other_peoples_code.md [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
+A great part of programming deals with people having to work with other people's code. It's part of the work trying to improve existing codebase and sometimes that involves a complete re-write.
 
 Take note from the great masters of programming, read the book [Coders at work - Reflections on the Craft of Programming](http://codersatwork.com/).
 
 Some of the largest and most successful codebases in world are codebases that has been developed by hundreds of people who have never even met each other, codebases developed without the use of any kind of framework, codebases done entirely in a procedural programming language without the use of anything but the procedural paradigm, and they wouldn't dream of doing it differently.
 
-The [Linux Kernel](https://www.kernel.org/) consist of more than 20 million lines of code all written entirely using procedural programming by more than 14.000 participants without the usage of any kind of framework.
+The [Linux Kernel](https://www.kernel.org/) consists of more than 20 million lines of code all written entirely using procedural programming by more than 14,000 participants without the use of any kind of framework.
```
Comment (QWp6t) [inline sections/en/being_afraid_of_other_peoples_code.md]: While a comma (`,`) is the most common in the English speaking world, a thin space (` `) is the _de facto_ international standard for digit grouping. The full stop (`.`) is common throughout non-English-speaking Europe.

Just throwing it out that his use of the full stop isn't wrong, so much as it's simply uncommon in English-speaking countries.

Comment (nleduc) [reply sections/en/being_afraid_of_other_peoples_code.md]: Good to know

**Comment by unixdigest** [comment]
Thank you very much!

**Pull request merged**